src/Symfony/Bridge/Doctrine/Security/RememberMe/DoctrineTokenProvider.php

```diff
@@ -53,11 +56,10 @@ public function __construct(Connection $conn)
     public function loadTokenBySeries(string $series)
     {
         // the alias for lastUsed works around case insensitivity in PostgreSQL
-        $sql = 'SELECT class, username, value, lastUsed AS last_used'
-            .' FROM rememberme_token WHERE series=:series';
+        $sql = 'SELECT class, username, value, lastUsed AS last_used FROM '.self::TABLE.' WHERE series=:series';
```

Comment:
I'd rather keep the SQL query as a raw literal rather than concatenating a private constant in it (which won't make it configurable anyway). That makes inspecting SQL queries easier.